save-frontend/src/main/kotlin/org/cqfn/save/frontend/App.kt

```diff
@@ -79,91 +79,93 @@ class App : RComponent<PropsWithChildren, AppState>() {
                             attrs {
-                                path = arrayOf("/:owner/:name/history")
-                                exact = true
-                                render = { routeResultProps ->
-                                    buildElement {
+                                path = "/:owner/:name/history"
+                                element = buildElement {
+                                    child(withRouter { _, params ->
                                         child(HistoryView::class) {
-                                            attrs.name = routeResultProps.match.params["name"]!!
-                                            attrs.owner = routeResultProps.match.params["owner"]!!
+                                            attrs.name = params["name"]!!
+                                            attrs.owner = params["owner"]!!
                                         }
-                                    }
+                                    })
                                 }
                             }
                         }
 
                         Route {
                             attrs {
-                                path = arrayOf("/:owner/:name/history/execution/:executionId")
-                                exact = true
-                                render = { props ->
-                                    buildElement {
+                                path = "/:owner/:name/history/execution/:executionId"
+                                element = buildElement {
+                                    child(withRouter { location, params ->
                                         child(ExecutionView::class) {
-                                            attrs.executionId = props.match.params["executionId"]!!
-                                            attrs.status = URLSearchParams(props.location.search).get("status")?.let(
+                                            attrs.executionId = params["executionId"]!!
+                                            attrs.status = URLSearchParams(location.search).get("status")?.let(
                                                 TestResultStatus::valueOf
                                             )
                                         }
-                                    }
+                                    })
                                 }
                             }
                         }
 
                         Route {
                             attrs {
-                                path =
-                                        arrayOf("/:owner/:name/history/execution/:executionId/details/:testSuiteName/:pluginName/:testFilePath+")
-                                exact = false  // all paths parts under testFilePath should be captured
+                                path = "/:owner/:name/history/execution/:executionId/details/:testSuiteName/:pluginName/:testFilePath/*"
```

Comment:
react-router v6 has dropped support for regexes in patterns, but trailing `*` now indicates that the entire ending of path should be captured. I tested it manually and it works as expected.